go/apps/api/openapi/spec/paths/chproxy/metrics/ChproxyMetricsRequestBody.yaml

```
@@ -0,0 +1,107 @@
+    city:
+      type: string
+      description: Client city
+      example: "New York"
+    colo:
+      type: string
+      description: Cloudflare colo/edge location
+      example: "EWR"
+    continent:
+      type: string
+      description: Client continent code
+      example: "NA"
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)